s3fs/core.py

```diff
@@ -858,7 +858,8 @@ async def _find(self, path, maxdepth=None, withdirs=None, detail=False, prefix="
 
     find = sync_wrapper(_find)
 
-    async def _mkdir(self, path, acl="private", create_parents=True, **kwargs):
+    async def _mkdir(self, path, acl=None, create_parents=True, **kwargs):
+        acl = acl or self.s3_additional_kwargs.get("ACL", "private")
```

Comment:
The bucket ACLs and key ACLs aren't quite the same sets, so this could be invalid ("private" is fine)